nginx-configs/image_server_to_iiif.js

```diff
@@ -50,7 +50,7 @@ function mapImageServerToIIIF(request, response) {
   // var imageUrl = "http://172.17.0.1:8182/iiif/2/" + identitifier + "/" + crop + "/" + urlSegment +"/0/default.jpg"
   
   //  When DR-1470 is complete, delegates.rb will need to know what derivative type is being requested.
-  if (paramsHash['t'] == 'u' || paramsHash['t'] == 'j' || paramsHash['t'] == 's') {
+  if (paramsHash['t'] == 'g' || paramsHash['t'] == 'j' || paramsHash['t'] == 's') {
```

Comment:
Feel like we can get rid of passing the type to the native server if we're parsing the request params instead.